data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -192,6 +197,16 @@ protected boolean shouldKeep(T item) {
     return remainingRowsFilter.filter(records);
   }
 
+  public Predicate<T> eqDeletedRows() {
```

Comment:
I'm a little confused with this naming, since I would assume "Rows" would return rows and this returns a filter basically